decred/decred/dcr/txscript.py

```diff
@@ -1790,6 +1790,16 @@ def payToStakeSHScript(addr, stakeCode):
 
 
 def multiSigScript(addrs, nRequired):
+    """
+    Create a multisig script for nRequired of addrs.
+
+    Args:
+        addrs (AddressSecpPubKey): Addresses that make up the script.
```

Comment:
I'd actually prefer to write the `Address` types like this, without the module. It's probably better to have the module, but I won't call these out because they read so much better.